src/app/vulnerabilities/Drupal Username Enumeration.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Factory\AppFactory;
+
+$app->get(
+    '/user/1',
+    function (Request $request, Response $response) {
+        $html = file_get_contents(__DIR__ . "/../resources/user/admin.html");
```

Comment:
Since this endpoint is just responsible for redirecting, this importion and rendering can be removed.

Reply:
Done.